<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LogWindow</class>
 <widget class="QDialog" name="LogWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1040</width>
    <height>597</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Logs</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Evolver log</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="check_log_evolver">
       <property name="text">
        <string>Log</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="check_error_evolver">
       <property name="text">
        <string>Error</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="push_evolver_clear">
       <property name="text">
        <string>Clear</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="check_clear_evolver">
       <property name="text">
        <string>auto</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Server log</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="check_log_server">
       <property name="text">
        <string>Log</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="check_error_server">
       <property name="text">
        <string>Error</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="push_server_clear">
       <property name="text">
        <string>Clear</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="check_clear_server">
       <property name="text">
        <string>auto</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QListWidget" name="list_error_evolver"/>
   </item>
   <item row="1" column="1">
    <widget class="QListWidget" name="list_error_server"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
